app/models/conversation.rb

```diff
@@ -26,6 +26,10 @@ class Conversation < ApplicationRecord
 
   validate :users_uniqueness, on: :create
 
+  def other_user(user)
```

Comment:
This is a little bit ambiguous, maybe recipient_user, or doesn't make any sense here?